jekyll/_cci2/api-permissions.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: classic-docs
+title: "API Permissions"
```

Comment:
Using API Permissions? Setting API Permissions? Title should start with action verb.

Reply:
I think "Using" applies more to the "Next Steps" section. "Setting" seems fine, but what do you think about "Managing"?